### PR TITLE
Layout fixes, buttons, placer of instructions and so on

### PR DESCRIPTION
**Pull request by jejja**
…d fonts

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/.idea
```
Comment (Tobbe) [inline .gitignore:1]: Please remove the .gitignore file from this pull request.
I want this pull request to focus only on the looks of the web page

File: index.html [hunk 2]
```diff
@@ -14,31 +16,33 @@
...
-            robot.openChest();<br>
-            robot.hasRedKey();<br>
-            robot.hasGreenKey();<br>
-            robot.hasBlueKey();<br>
-            if (robot.has&lt;Color&gt;Key()) {<br>
-            &nbsp;&nbsp;&nbsp;&nbsp;robot.&lt;SomeMethod&gt;();<br>
-            }
-            robot.openDoor();
-        </code>
+        <div class="info-area">
```
Comment (Tobbe) [inline index.html]: This `<div>` is not needed. Just put the class directly on the `<code>` element instead

File: index.html [hunk 3]
```diff
@@ -50,10 +54,12 @@
             For example, the following program would make the robot walk two
             steps to the right, and then one step up.
         </p>
-        <code>
-            robot.moveRight(2);<br>
-            robot.moveUp();
-        </code>
+        <div class="info-area">
```
Comment (Tobbe) [inline index.html]: Same here, div not needed

File: style.css [hunk 4]
```diff
@@ -1,3 +1,16 @@
+
+h1 {
+    font-family: ‘Metrophobic’, Arial, serif; font-weight: 400;
+}
+
+p {
+    font-family: ‘Metrophobic’, Arial, serif; font-weight: 400;
```
Comment (Tobbe) [inline style.css]: No need to duplicate this, just make the h1 rule apply to p elements as well `h1, p { ... }`

File: style.css [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+}
+
+input[type=button] {
```
Comment (Tobbe) [inline style.css]: We only have one kind of inputs for now, so you can remove `[type=button]`

File: style.css [hunk 6]
```diff
@@ -33,9 +46,19 @@ input {
 
 .instructions {
     position: absolute;
-    top: 470px;
+    top: 0px;
```
Comment (Tobbe) [inline style.css]: You shouldn't specify units for `0` (`0` is the same in all units :wink:)

File: style.css [hunk 7]
```diff
@@ -1,3 +1,16 @@
+
+h1 {
+    font-family: ‘Metrophobic’, Arial, serif; font-weight: 400;
```
Comment (Tobbe) [inline style.css:2]: Please write multiple css rules on separate lines (i.e. move the font-weight to its own line)
Actually, isn't 400 the default? I think you can remove it all together

File: style.css [hunk 8]
```diff
@@ -33,9 +46,19 @@ input {
...
+    margin-left: 10px;
+    margin-right: 10px;
 }
 
 .field_item {
     position: absolute;
 }
+
+.info-area {
+    background-color: rgb(252,221,255);
```
Comment (Tobbe) [inline style.css]: I don't know why, but for some reason I prefer hex values. Unless you have a good point for using rgb() notation, could you please convert it to hex? Thanks

**Comment by Tobbe** [comment]
Please shorten the title of your commit message so that the full title is displayed in GitHub without the ...

If you want to proved more info, do so in the body of the message (i.e. after the title plus an empty line).

File: .gitignore (hunk 1 above)
Comment (jejja) [inline .gitignore:1]: Yeah, I understand your point. But in this case I think its not important. ;)

**Comment by jejja** [comment]
Fixed most of the comments.

**Comment by jejja** [comment]
Sådär. Mergad med din branch. Bara att ta in.

**Pull request closed without merging**